huaweicloud/services/acceptance/acceptance.go

```diff
@@ -196,14 +196,56 @@ func TestCheckResourceAttrWithVariable(resourceName, key, varStr string) resourc
 		// Get the value based on resName and keyName from the state.
 		rs, ok := s.RootModule().Resources[resName]
 		if !ok {
-			return fmtp.Errorf("Can't find %s in state : %s.", resName, ok)
+			return fmtp.Errorf("Can't find %s in state : %v.", resName, ok)
 		}
 		value := rs.Primary.Attributes[keyName]
 
 		return resource.TestCheckResourceAttr(resourceName, key, value)(s)
 	}
 }
 
+func (rc *resourceCheck) getResourceCount(s *terraform.State) (int, error) {
+	var count int
+	for _, rs := range s.RootModule().Resources {
+		isMatch, err := regexp.MatchString(fmt.Sprintf("^%s\\.\\d+$", rc.resourceName), rs.Type)
+		if err != nil {
+			return count, err
+		}
+		if isMatch {
+			count++
+		}
+	}
+	return count, nil
+}
+
+func (rc *resourceCheck) checkResourceDestroy(s *terraform.State) error {
+	strs := strings.Split(rc.resourceName, ".")
+	var resourceType string
+	for _, str := range strs {
+		if strings.Contains(str, "huaweicloud_") {
+			resourceType = strings.Trim(str, " ")
+			break
+		}
+	}
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != resourceType {
+			continue
+		}
+
+		conf := TestAccProvider.Meta().(*config.Config)
```

Comment:
suggest to put this line outside for loop.

Reply:
done